src/lib/src/component/navigation/navigation.component.ts

```diff
@@ -151,7 +155,8 @@ export class NavigationComponent implements OnInit {
   }
 
   selectionIsExecutable(): boolean {
-    return this.uiState.selectedElement != null && this.uiState.selectedElement.path.endsWith('.tcl');
+    return this.uiState.selectedElement == null && this.uiState.activeEditorPath != null && this.uiState.activeEditorPath.endsWith('.tcl')
```

Comment:
just flavor: operator preceedance allows it, but I'd rather have parenthesis around the || options.
How about === null checks?